packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -1647,6 +1648,10 @@ describe('plugin-meetings', () => {
             .stub()
             .resolves({turnServerInfo: {}, turnDiscoverySkippedReason: 'reachability'});
 
+          meeting.sendSlotManager = new SendSlotManager({logger: {info: () => {}}});
```

Comment:
this code is inside the describe "addMedia/updateMedia semi-integration tests", these are not typical unit tests where everything is mocked. In these tests only external interfaces like internal-media-core are mocked, so we shouldn't be mocking the SDK's SendSlotManager, but instead mock only the createSendSlot() method of MultistreamRoapMediaConnection to return fake send slots and the tests should check that we call the right methods on these WCME send slots

Reply:
@marcin-bazyl - We do have [tests to fake the sendSlots in the SendSlotManager tests](https://github.com/webex/webex-js-sdk/blob/stream-classes/packages/%40webex/plugin-meetings/test/unit/spec/multistream/sendSlotManager.ts) anyway. That is why we just mock the sendSlotManager. Should we still do this?

Reply:
yes, I would rather not mock sendSlotManager in these tests.

Reply:
Done. Removed mocking of sendSlotManager and mocked Slots everywhere. Please check.